tests/acceptance/profile/navigate-to-one-of-latest-issue.feature

```diff
@@ -0,0 +1,21 @@
+@setupApplicationTest
+Feature: Profile | navigate to one of latest issue
+
+  Scenario: navigate to one of the latest issue of user
+
+    Given There is no pre-existing data
+    And default scenario is loaded
+    And User_1 is logged in
+    And User_1 selects Project 2
+    And Project has following details
+    ---------
+    | issue |
+    | 5     |
+    ---------
+    And User has following details
+    ---------------------------------------
+    | latestIssues(userlatestissue)       |
+    | 5                                   |
+    ---------------------------------------
+    When User navigates to app/user/1
+    When User clicks on latest issue 5
```

Comment:
There needs to be some verification for this to be a test case, there should be some then statements otherwise what are we testing?